src/rt/test/threadping.h

```diff
@@ -0,0 +1,87 @@
+// Copyright Microsoft and Project Verona Contributors.
+// SPDX-License-Identifier: MIT
+#include <condition_variable>
+#include <thread>
```

Comment:
Would it be possible to add support for VERONA_EXTERNAL_THREADING to this file?
```#if !defined(VERONA_EXTERNAL_THREADING)

#  include <thread>

namespace verona::rt
{
  using InternalThread = std::thread;
}

#else

#  include <verona_external_threading.h>

#endif
```

and using ```verona::InternalThread``` instead of ```std::thread```.